scripts/noshake.json

```diff
@@ -329,6 +329,8 @@
   "Mathlib.RingTheory.SimpleModule.Basic": ["Mathlib.Data.Matrix.Mul"],
   "Mathlib.RingTheory.PowerSeries.Basic":
   ["Mathlib.Algebra.CharP.Defs", "Mathlib.Tactic.MoveAdd"],
+  "Mathlib.RingTheory.PolynomialLaw.Basic":
```

Comment:
What happens if you keep `noshake.json` intact?

Reply:
I'll try.

Reply:
IIRC, `import Mathlib.Algebra.MvPolynomial.Basic`is considered as unnecessary. It was made necessary by an `open MvPolynomial`, but in its present state, that `open` line is not necessary.